### PR TITLE
Unable to scroll horizontally

### PR DESCRIPTION
**Pull request by AminTi**

### CODE REVIEW

File: src/pages/session/DeleteAllSessions.ts
Comment (madkoo) [inline src/pages/session/DeleteAllSessions.ts:1]: Rename ts file to DeleteSessions as it is only deleting a single sessions

File: src/services/channels/signalr.ts [hunk 1]
```diff
@@ -70,12 +70,13 @@ export class SignalRChannel implements IChannel {
             )
             .configureLogging(signalR.LogLevel.Information)
             .build();
-
+        console.log(this.connection, "test connection")
```
Comment (madkoo) [inline src/services/channels/signalr.ts:73]: is Logging needed ?

File: src/services/channels/signalr.ts [hunk 2]
```diff
@@ -70,12 +70,13 @@ export class SignalRChannel implements IChannel {
...
         // Hook up handler for all messages the server sends
-        this.connection.on("broadcast", this.onReceive);
+        this.connection.on("broadcast", this.onReceive); 
 
         // Start connection
         await this.connection.start().catch(err => {
+            console.log(this.connection, "test connection 2")
```
Comment (madkoo) [inline src/services/channels/signalr.ts:79]: is Logging needed ?

File: src/services/channels/signalr.ts [hunk 3]
```diff
@@ -86,7 +87,7 @@ export class SignalRChannel implements IChannel {
             name: identity.displayName,
             imageUrl: identity.imageUrl
         });
-
+        console.log(this.connection, "test connection 3")
```
Comment (madkoo) [inline src/services/channels/signalr.ts:90]: is Logging needed ?

File: src/services/channels/signalr.ts [hunk 4]
```diff
@@ -103,6 +104,7 @@ export class SignalRChannel implements IChannel {
     }
 
     private onReceive = (action: Action, payload: any) => {
+        console.log("test")
```
Comment (madkoo) [inline src/services/channels/signalr.ts:107]: is Logging needed ?

File: src/components/votes.tsx [hunk 5]
```diff
@@ -24,7 +24,7 @@ export class Votes extends React.Component<IVotesProps> {
 
       
     return (
-            <div className="flex-row">
+            <div className="flex-row" style={{display: "flex", flexDirection: "row", flexWrap:"wrap"}}>
```
Comment (madkoo) [inline src/components/votes.tsx:27]: I would prefer custom styling to be in with its own class name in a scss/css file

**Review by madkoo — COMMENTED** [review]
Take a look at my comments :)

One things is confusing, the title of the PR is "unable to scroll horizontally" but I see you are changing the session deletion logic here ??

**Comment by AminTi** [comment]
I will close this branch and create new  clean one from main

**Pull request closed without merging**